src/common/tests/SConscript

```diff
@@ -19,8 +21,12 @@ def scons():
                     LIBS=['daos_common', 'gurt', 'pmemobj', 'cmocka'])
     daos_build.test(denv, 'other', 'other.c',
                     LIBS=['daos_common', 'gurt', 'cart'])
-    daos_build.test(denv, 'checksum', 'checksum.c',
-                    LIBS=['daos_common', 'gurt', 'cart'])
+    daos_build.test(denv, 'common_test_runner',
+    			['common_test_runner.c', 'daos_checksum_tests.c',
+    			'misc_tests.c'],
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (add 1 space).

Reply:
you can run ./utils/check_modules -s src/common/tests/SConscript locally